kerberos.json

```diff
@@ -0,0 +1,126 @@
+{
+  "services": [
+    {
+      "name": "CDAP",
+      "identities": [
+        {
+          "name": "/spnego"
+        },
+        {
+          "name": "/hdfs"
+        },
+        {
+          "name": "cdap",
+          "principal": {
+            "value": "cdap@${realm}",
+            "type" : "user",
+            "configuration": "cdap-env/cdap_principal_name",
+            "local_username": "cdap"
+          },
+          "keytab": {
+            "file": "${keytab_dir}/cdap.headless.keytab",
```

Comment:
Yeah, so most people don't like using a service principal for non-service work. Even doing something like `hive -e 'set -v'` should be done using this headless user, versus the service user. This allows someone to restrict the two differently in their security policy implementation.